src/api/pricer/CoinBase.ts

```diff
@@ -38,17 +38,13 @@ export default class CoinBase extends BaseExchangeApi {
         `${URL}?currency=${sourceCurrency}`
       );
     } catch (error: unknown) {
-      const axiosError = error as AxiosError;
-      const { response } = axiosError;
-      if (response) {
-        throw Error(
-          `CoinBase API status ${response.status}/${response.statusText}`
-        );
-      }
-      if (axiosError.request) {
-        throw Error('No response received from CoinBase API');
+      const { response } = error as ResponseError;
```

Comment:
I'd log the error here, otherwise we won't be able to understand where it comes from, since we're generating a new error.

Reply:
I think the reason it was not logged to begin with is that the interceptor for the wrapper will log it on error. Let me know if I should still log this though.